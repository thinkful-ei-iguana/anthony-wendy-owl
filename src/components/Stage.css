.stage {
  display: flex;
  justify-content: space-around;
  width: 1000px;
  background: slategray;
}

.img-stage {
  padding-top: 100px;
}
