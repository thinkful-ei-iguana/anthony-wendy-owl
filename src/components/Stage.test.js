import React from "react";
import ReactDOM from "react-dom";
import renderer from "react-test-renderer";
import Stage from "./Stage";

describe("Participants component", () => {
  it("renders without crashing", () => {
    const div = document.createElement("div");
    ReactDOM.render(<Stage />, div);
    ReactDOM.unmountComponentAtNode(div);
  });
  it("renders the UI as expected", () => {
    const tree = renderer
      .create(<Stage participants={["test person", "another person"]} />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
