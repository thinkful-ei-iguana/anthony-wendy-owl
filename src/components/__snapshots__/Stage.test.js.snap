// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Participants component renders the UI as expected 1`] = `
<div
  className="stage"
/>
`;
